services/blockchain-indexer/shared/dataService/business/interoperability/blockchainApps.js

```diff
@@ -71,11 +73,24 @@ const getBlockchainApps = async (params) => {
 
 	const total = await blockchainAppsTable.count(params);
 
-	blockchainAppsInfo.data = await blockchainAppsTable.find(
+	const resultSet = await blockchainAppsTable.find(
 		{ ...params, limit: params.limit || total },
 		Object.getOwnPropertyNames(blockchainAppsTableSchema.schema),
 	);
 
+	blockchainAppsInfo.data = await BluebirdPromise.map(
+		resultSet,
+		async blockchainAppInfo => {
+			const { escrowedAmounts } = await requestConnector('getEscrowedAmounts');
```

Comment:
The escrowedAmount must be queried from a chain corresponding the chainID. We'll need to query the ServiceURLs from the app-registry and fetch this information.

Reply:
Please ignore this comment!